app/views/candidate_interface/decisions/offer.html.erb

```diff
@@ -30,10 +30,10 @@
       <p class="govuk-body">If you need help, you can speak to our <%= govuk_link_to 'teacher training advisers', t('get_into_teaching.url_get_an_advisor') %>.</p>
 
       <%= f.govuk_radio_buttons_fieldset :response, legend: { text: t('decisions.response.legend'), size: 'm' } do %>
-        <% if @application_choice.offer&.dig('conditions')&.present? %>
-          <%= f.govuk_radio_button :response, 'accept', label: { text: t('decisions.response.accept.label') }, link_errors: true %>
-        <% else %>
+        <% if @application_choice.unconditional_offer? %>
```

Comment:
I don't think we can use this method here because of the `return false unless recruited?` line.